plugins/moxiemanager.md

```diff
@@ -6,14 +6,12 @@ description: File and image management plugin and service
 keywords: amazon azure premium pro enterprise tiny relative_urls
 ---
 
-{% include pardot-px-moxiemanager.html %}
+MoxieManager is a premium {{site.productname}} plugin and server system that enables users to insert files located externally to the editor (e.g. on their client desktop) into their document.
 
-MoxieManager is a premium TinyMCE plugin and server system that enables users to insert files located externally to the editor (e.g. on their client desktop) into their document.
+MoxieManager is an application separate from {{site.productname}} that includes .NET or PHP server components. Being built by the team behind {{site.productname}} it has tight integration with the editor via a toolbar control. It is designed to work with {{site.productname}} configuration settings such as `relative_urls`.
 
-MoxieManager is an application separate from TinyMCE that includes .NET or PHP server components. Being built by the team behind TinyMCE it has tight integration with the editor via a toolbar control. It is designed to work with TinyMCE configuration settings such as `relative_urls`.
+MoxieManager is a paid addition to {{site.productname}}. It is included in [{{site.accountpage}} subscriptions]({{site.pricingpage}}).
```

Comment:
Should this be `{{accountpage}} subscriptions` or `{{sfenterpriseac}}`?

Reply:
updated